static/js/upload_widget.js

```diff
@@ -39,8 +39,7 @@ var upload_widget = (function () {
 
         function clear() {
             var control = get_file_input();
-            var new_control = control.clone(true);
-            control.replaceWith(new_control);
+            control.val('');
```

Comment:
When I get an error from upload widget, the widget doesn't clear file input value at this place. And after submitting it sends a wrong file to the backend. So replacing element doesn't work. That's why I've changed it to clean value. I will move it to separate commit.